Essentials/src/com/earth2me/essentials/MetaItemStack.java

```diff
@@ -315,7 +318,9 @@ public void addFireworkMeta(final CommandSource sender, final boolean allowShort
                 String[] colors = split[1].split(",");
                 for (String color : colors) {
                     if (colorMap.containsKey(color.toUpperCase())) {
-                        fadeColors.add(colorMap.get(color.toUpperCase()).getFireworkColor());
+                        fadeColors.add(colorMap.get(color.toUpperCase()).getColor());
```

Comment:
Could you explain why you changed the call `getFireworkCaller()` to `getColor()`?